packages/web-integration/src/common/agent.ts

```diff
@@ -55,7 +56,7 @@ export class PageAgent {
       executions: [],
     };
     this.taskExecutor = new PageTaskExecutor(this.page, {
-      cache: opts?.cache || { aiTasks: [] },
+      testFilePath: opts?.testFilePath,
```

Comment:
Should we name it as `cacheContext` ? `PageTaskExecutor` should not be coupled with the test file path